blog/_posts/2023-04-18-integrating-scalajs-with-vite.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: blog-detail
+post-type: blog
+by: Sébastien Doeraene
+title: Integrating Scala.js with Vite
+description: Using the new vite-plugin-scalajs to better integrate Scala.js with the Vite build tool.
+---
+
+One of the cornerstones of Scala.js is its ability to interoperate with JavaScript, in order to leverage its massive ecosystem.
+At the language level, we have good foundations and good tooling (namely [ScalablyTyped](https://scalablytyped.org/)) to reach out to JavaScript libraries.
```

Comment:
“reach out” means communication between people — sounds odd in relation to libraries. not sure what the best substitute is. maybe “to make use of” or “for seamless interop with” or something more explicit like “to use JavaScript libraries from Scala.js projects"